src/gxcore/common/geolocation.ts

```diff
@@ -0,0 +1,241 @@
+import { notImplemented } from "../../misc/helpers";
+
+export class GeneXusCommonGeolocation {
+
+  /**
+   * Returns the current location for the device
+   * @param minAccuracy
+   * @param timeout
+   * @param includeHeadingAndSpeed
+   * @param ignoreErrors
+   * @return any
+   */
+  public static getMyLocation(minAccuracy: number, timeout: number, includeHeadingAndSpeed:boolean, ignoreErrors:boolean): Promise<GeneXusCommonGeolocationInfoData> {
+    return new Promise<GeneXusCommonGeolocationInfoData>((resolve, reject) => {
+      if (navigator.geolocation) {
```

Comment:
I think there should be an `else` to handle the case where `navigation.geolocation` is not available.